libraries/cmake/source/icu/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+function(icuMain)
```

Comment:
The Thrift library makes use of the Boost's locale library, which in turn will include some of the ICU headers. It doesn't seem like we actually need to build the whole library, but further testing is appreciated to make sure everything's good.

In case we do end up building this, it depnds on the `icu-le-hb` library that is not yet imported

Reply:
I guess this can stay like this for now